src/FluentlyHttpClient/HttpRequestMessageExtensions.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Net.Http;
+
+namespace FluentlyHttpClient
+{
+	internal static class HttpRequestMessageExtensions
+	{
+		internal static FluentHttpRequest ToFluentlyHttpRequest(
+			this HttpRequestMessage request,
+			FluentHttpClient client
+		)
+			=> new FluentHttpRequest(new FluentHttpRequestBuilder(client), request);
```

Comment:
in order to create a new instance `FluentHttpRequestBuilder` use the factory, as it sets some defaults etc... + could be less breaking.

`client.CreateRequest` + ideally set several data from request to the builder